ilc/server/registry/Registry.js

```diff
@@ -52,6 +52,18 @@ module.exports = class Registry {
             responseType: 'json',
         });
 
+        //FIXME: remove TEMP code:
```

Comment:
can we remove it now?

Reply:
see https://github.com/namecheap/ilc/pull/249/files#diff-569e299812f8148a25f8ef50aaa9f5cab818b6395ffcbb9310658a3f4b49074e